evan_did_method_spec.md

```diff
@@ -64,14 +67,13 @@ Asset identities, digital twins, or simply assets, are created and owned by a SS
 The following sample shows an asset identity's DID document. The asset has no `authentication` means and only points to its owner's SSI which is delegated to authenticate in this asset identity's name, as well as apply changes to the DID document (via the `controller` property).
 
 ```js
-{
-    '@context': 'https://w3id.org/did/v1',
-    id: 'did:evan:testcore:0x071be9398b832d711b2e6278eba372ec968d8073e33a9cac9289c3ed06fc0701',
-    controller: 'did:evan:testcore:0x126E901F6F408f5E260d95c62E7c73D9B60fd734',
-    authentication: [
-        'did:evan:0xb9A7884DEf4F8E503C50068dCB03d2393cBDFA33#key-1'
-    ],
-    service: []
+{ '@context': 'https://w3id.org/did/v1',
+  id:
+   'did:evan:testcore:0x8dfe627ab531a9b9b23910ca691df53dfb13c1b157777360d53d68172aa30ba9',
```

Comment:
I cannot resolve this DID at [https://testcore.evan.network/did/did:evan:testcore:0x8dfe627ab531a9b9b23910ca691df53dfb13c1b157777360d53d68172aa30ba9](the resolver). Does ist exist?